### PR TITLE
Refactor/AWT-63_Third_person_steps

### PR DESCRIPTION
**Pull request by d4vidmc**
Implementing steps to match existing steps implementation and some changes needed to match new steps definition
AssertSteps intends to concentrate some common steps validation.
ToastMessage Intends to capture the message on screen and validate an expecting message.

### CODE REVIEW

File: src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/WorkspaceSteps.java [hunk 1]
```diff
@@ -56,39 +56,46 @@ public void setWorkSpacename(final String strname) {
...
-        assertTrue(this.dashboard.existWorkSpace(name));
-    }
-
-    /** This step create a workspace. **/
+    /**
+     * This step create a workspace.
+     **/
     @Given("create an workspace")
-    public void createaworkspace() {
+    public static void createaworkspace() {
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/WorkspaceSteps.java:76]: Why the step definition is static ?

File: src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/TaskSteps.java [hunk 2]
```diff
@@ -33,114 +37,125 @@
 
     @Autowired
     private Tasks tasksPanel;
+    @Autowired
+    private SoftAssert softAssert;
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/TaskSteps.java]: Could you update this PR according to the things that we already talked before?

**Review by carledriss — CHANGES_REQUESTED** [review]

File: src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/ProjectSteps.java [hunk 3]
```diff
@@ -77,8 +77,8 @@ public void openProjectsSettings() {
      */
     @When("user click over delete project link")
     public void userClickOverDeleteProjectLink() {
-        this.confirm.clickOnDeleteProjectLink();
-        this.confirm.clickOnDeleteButton();
+        this.confirm.clickOnDeleteWorkspaceProjectLink();
```
Comment (rrchcr8) [inline src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/ProjectSteps.java:80]: this interferes with my branch

File: src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/WorkspaceSteps.java (hunk 1 above)
Comment (d4vidmc) [reply src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/WorkspaceSteps.java:76]: I do not know how this come to this, may due to a merge from develop, anyway it should not appear again

File: src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/TaskSteps.java (hunk 2 above)
Comment (d4vidmc) [reply src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/TaskSteps.java]: `    /**
     * Based on tag annotation enable soft assert.
     */
    @Before("@SoftAssert")
    public static void initialize() {
        softAssert = new SoftAssert();
    }

    /**
     * Final step validation for soft assert.
     */
    @And("asserts all")
    public static void assertAll() {
        softAssert.assertAll();
    }`

**Review by rrchcr8 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**